src/components/Kebab/Kebab.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import kebabs from '../../mocks/kebab.json'
+import Product from '../Products/Product'
+import PropTypes from 'prop-types'
+import Headline from '../Headline/Headline'
+
+
+export default function Kebab(){
+    return(
+        <>
+        <Headline headline="Kebab" />
+        {kebabs.map((kebab, index) => (
+        <Product key={index} title={kebab.title} image={kebab.image} />
+        ))}
+        </>
+    )
+}
+
+Kebab.propTypes = { 
```

Comment:
I'm not sure if the proptypes are correct. <Kebab /> doesn't receive any props. Only <Product /> gets some props.